package.json

```diff
@@ -71,5 +71,10 @@
     "@grpc/proto-loader": "0.5.0",
     "big.js": "5.2.2",
     "grpc": "1.19.0"
+  },
+  "standard": {
+    "env": [
+      "mocha"
```

Comment:
@clark800 was your editor giving you issues? We might not have updated the docs saying we dont use standard anymore. The plugin would be eslint

Reply:
Ok, is there a custom eslintrc file or are you just using the eslint default configuration?

Reply:
@dannypaz updated